javascript/javascript2/week1/spiritanimalg.js

```diff
@@ -0,0 +1,25 @@
+const spiritAnimal = [
+  "Confident Bear",
+  "Competitive Wolf",
+  "Introverted Panda",
+  "Graceful Butterfly",
+  "Powerful Horse",
+  "Mischivous Fox",
+  "Strongwilled Tiger",
+  "Couregues Lion",
+  "Loving Dolphin",
+  "Optimistic Hawk",
+];
+
+const inputName = document.getElementById("name");
```

Comment:
You should also pick a better name for these elements, naming a div with id input can be really confusing for other people reading your code, you could try input container for example.